packages/lit-dev-content/site/tutorials/content/working-with-lists/00.md

```diff
@@ -0,0 +1,13 @@
+In this tutorial, we will be exploring different ways of rendering a list of
+items in Lit!
+
+Let's begin by looking a how we might render a list of names from an array.
```

Comment:
Let's not use` Let's` who is the implied `we`? is it the Lit team, the user and us, is it the Lit library itself?

Suggestion:

> Begin my rendering a list of names from an array. The example already has set up a custom element with the `names` property that is an array of strings.

some help:

- commands: just give the command
  - lets begin -> begin
  - we can see -> "See" or "You may notice" or plainly "Notice"
    - prefer the assertive voice rather than "may" so the user can be assured they know that we know what we're teaching them
- we
  - tell the user who we is
  - we -> the example / the code